libsignal-service/Cargo.toml

```diff
@@ -12,14 +12,13 @@ license = "GPLv3"
 readme = "../README.md"
 
 [dependencies]
-libsignal-protocol = { git = "https://github.com/signalapp/libsignal-client", tag = "v0.17.0" }
-zkgroup = { git = "https://github.com/signalapp/libsignal-client", tag = "v0.17.0" }
+libsignal-protocol = { git = "https://github.com/signalapp/libsignal-client", tag = "v0.20.0" }
+zkgroup = { git = "https://github.com/signalapp/libsignal-client", tag = "v0.20.0" }
```

Comment:
@boxdot
Apparently, `https://github.com/signalapp/libsignal-client` now redirects to `https://github.com/signalapp/libsignal`.
Could you update this too, please?